nomad/structs/structs.go

```diff
@@ -11040,6 +11040,16 @@ type ACLTokenUpsertResponse struct {
 	WriteMeta
 }
 
+// OneTimeToken is used to log into the web UI using a token provided by the
+// command line.
+type OneTimeToken struct {
+	OneTimeSecretID string
```

Comment:
Note for reviewers: I changed the name `OneTimeSecret` field from the RFC to `OneTimeSecretID` so as to be more consistent with how we name similar fields with our ACL tokens.